_posts/2023-10-04-release.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Release 2023.40
+author: Priya
+date: 2023-10-04
+category: Jekyll
+layout: post
+---
+
+### Defect Fixes:
+* Resolved an issue where Reports couldn't be opened when the user wasn't subscribed to any project.
```

Comment:
Make R in reports small case to be consistent with usage elsewhere in the release notes

Reply:
fixed